tests/test-iso-kargs.sh

```diff
@@ -25,6 +26,14 @@ fi
 # Test all the modification operations.
 coreos-installer iso kargs modify -a foobar=oldval -a dodo "${iso}"
 coreos-installer iso kargs show "${iso}" | grep -q 'foobar=oldval dodo'
+# Stream modification to stdout.
+coreos-installer iso kargs modify -a foobar=oldval -a dodo -o - "${iso}" > "${stdout_iso}"
+coreos-installer iso kargs show "${stdout_iso}" | grep -q 'foobar=oldval dodo'
```

Comment:
This check will pass in any case, since the previous stanza already added these arguments to `${iso}`.  Since these checks don't depend on the surrounding modifications of `${iso}`, it'd make sense to move them.

Reply:
:man_facepalming: thanks